### PR TITLE
MGMT-9691: Allow overriding hypershift control plane operator image

### PR DESCRIPTION
**Pull request by eranco74**
[MGMT-9691](https://issues.redhat.com/browse/MGMT-9691): CAPI e2e test is failing due to hypershift control plane operator image mismatch
 
 [MGMT-9213](https://issues.redhat.com/browse/MGMT-9213): Add test_prepare_late_binding_kube_api_ipv4_highly_available to test_kube_api
 This can be used by capi test for setting up the env

Save hypershift logs to file

### CODE REVIEW

File: src/tests/test_kube_api.py [hunk 1]
```diff
@@ -195,6 +195,7 @@ def capi_test(
                     pull_secret_file=ps,
                     agent_namespace=spoke_namespace,
                     provider_image=os.environ.get("PROVIDER_IMAGE", ""),
+                    hypershift_CPO_image=os.environ.get("HYPERSHIFT_IMAGE", ""),
```
Comment (osherdp) [inline src/tests/test_kube_api.py:198]: small preference for hypershift_cpo_image

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by eranco74** [comment]
/override ci/prow/e2e-metal-assisted-kube-api-late-binding-single-node
Not related to changes in this PR

**Pull request merged**